packages/navstar/build

```diff
@@ -22,7 +22,7 @@ mkdir -p $(dirname $service)
 
 navstar_service=${PKG_PATH}/dcos.target.wants/dcos-navstar.service
 mkdir -p $(dirname $navstar_service)
-cp /pkg/extra/dcos-navstar.service "$navstar_service"
+envsubst '$PKG_PATH' < "/pkg/extra/dcos-navstar.service" > "${navstar_service}"
```

Comment:
The sections on ExecStart, etc. specifically mention when they support environment variable substitution: https://www.freedesktop.org/software/systemd/man/systemd.service.html